.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: 'CI'
+# **What it does**: Runs test on the CI.
```

Comment:
**suggestion (optional)**

I didn't think the comment provided helpful information. It may be better to delete the line to keep the file simple.

```suggestion
```

Reply:
Done, thanks!